index.js

```diff
@@ -1,66 +1,32 @@
-const fs = require("fs-extra");
-const path = require("path");
+const path = require('path');
 
-const getNetlifyCacheDirs = ({ config, constants }) => {
-  const cacheDir = constants.CACHE_DIR;
-  const gatsbyDir = path.dirname(config.build.publish);
-
-  return {
-    gatsbyCacheDir: path.join(gatsbyDir, ".cache"),
-    gatsbyPublicDir: path.join(gatsbyDir, "public"),
-    netlifyCacheDir: path.join(cacheDir, "gatsby/.cache"),
-    netlifyPublicDir: path.join(cacheDir, "gatsby/public")
-  };
-};
+const getCacheDirs = constants => [
+  constants.BUILD_DIR,
+  path.normalize(`${constants.BUILD_DIR}/../.cache`),
+];
 
 module.exports = () => {
   return {
-    name: "netlify-plugin-gatsby-cache",
-    onPreBuild: async args => {
-      const {
-        gatsbyCacheDir,
-        gatsbyPublicDir,
-        netlifyCacheDir,
-        netlifyPublicDir
-      } = getNetlifyCacheDirs(args);
+    name: 'netlify-plugin-gatsby-cache',
+    async onPreBuild({ constants, utils }) {
+      const cacheDirs = getCacheDirs(constants);
 
-      if (!fs.existsSync(netlifyCacheDir) || !fs.existsSync(netlifyPublicDir)) {
-        console.log("No Gatsby cache found. Building fresh...");
-        return;
-      }
+      console.log({ cacheDirs });
```

Comment:
I am assuming this might have been left for debugging and is meant to be removed @jlengstorf ?